addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1971,8 +1971,15 @@ export class OdooEditor extends EventTarget {
         // Let the DOM split and delete the range.
         const contents = range.extractContents();
 
-        setSelection(start, nodeSize(start));
         const startLi = closestElement(start, 'li');
+        const endli = closestElement(end, 'li');
+        let shouldMergeLi;
```

Comment:
Let's rather name it `keepEndLi`.

Reply:
I meant to change `shouldMergeLi` to `keepEndLi`, the reason I want it to be changed because you change `doJoin` to false which means you don't want to merge to li's but then there is another variable `shouldMergeLi` which is true meaning you want to join the lists. Sounds confusing :confused:.